samples/stc/stctest.cpp

```diff
@@ -672,7 +672,8 @@ AppAbout::AppAbout (wxWindow *parent,
     }
 
     // Get version of Scintilla
-    wxVersionInfo vi = wxStyledTextCtrl::GetLibraryVersionInfo();
+    wxVersionInfo vi1 = wxStyledTextCtrl::GetLibraryVersionInfo();
+    wxVersionInfo vi2 = wxStyledTextCtrl::GetLexerVersionInfo();
```

Comment:
I think we should make the names more readable.
```suggestion
    wxVersionInfo viScintilla = wxStyledTextCtrl::GetLibraryVersionInfo();
    wxVersionInfo viLexilla = wxStyledTextCtrl::GetLexerVersionInfo();
```